test/e2e/addon_update.go

```diff
@@ -283,6 +284,7 @@ var _ = framework.KubeDescribe("Addon update", func() {
 		sshExecAndVerify(sshClient, fmt.Sprintf("sudo cp %s/%s %s/%s", temporaryRemotePath, svcv2, destinationDir, svcv2))
 		sshExecAndVerify(sshClient, fmt.Sprintf("sudo rm %s/%s", destinationDir, rcv1))
 		sshExecAndVerify(sshClient, fmt.Sprintf("sudo rm %s/%s", destinationDir, svcv1))
+		sshExecAndVerify(sshClient, fmt.Sprintf("sudo pkill -9 -o -f kube-addon"))
```

Comment:
Yes we do if we don't want it to take the recheck interval (15 minutes) for an update to be propagated. The kube-addon-manager will recheck when it is restarted so this works well